nextstrain_profiles/nextstrain-open/builds.yaml

```diff
@@ -387,30 +246,18 @@ subsampling:
 
   # Custom subsampling logic for region Asia over 6m
   # Grouping by division
-  # Separating three buckets for China, India and elsewhere
+  # Grouping by country weighted by population size
   # 4375 total
```

Comment:
is this comment still correct?

Reply:
Yes, it's still 4375 total (700+175+2800+700). Note that this is the **target** size and doesn't reflect number of samples that end up in the Auspice dataset. That can be less due to the caveat in the PR description and other steps downstream of subsampling such as sequence length filter, clock filter, etc.

Reply:
thanks -- i was missing the 175 because of a code fold in the PR >_<